var/spack/repos/builtin/packages/py-tuswsgi/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class PyTuswsgi(PythonPackage):
+    """python wsgi filter for tus protocol 1.0.0"""
+
+    homepage = "https://github.com/mvdbeek/tusfilter"
+    url = "https://pypi.io/packages/py2.py3/t/tuswsgi/tuswsgi-0.5.4-py2.py3-none-any.whl"
+
+    version(
+        "0.5.4",
+        sha256="f681a386254a161a97301a67c01ee7da77419c007d9bc43dbd48d5a987491a5e",
+        expand=False,
+    )
+
+    depends_on("python@3.6:", type=("build", "run"))
+    depends_on("py-setuptools", type="build")
```

Comment:
If you're installing from a wheel, setuptools isn't actually needed because the package is already "built".

Reply:
done, thanks